purview/utils/general.py

```diff
@@ -105,3 +105,84 @@ def get_neo4j_node(resource_name, uid):
     error = ('The requested resource "{0}" is invalid. Choose from the following: '
              '{1}, and {2}.'.format(resource_name, ', '.join(model_names[:-1]), model_names[-1]))
     raise ValidationError(error)
+
+
+def node_query(node_label, uid_name=None, uid=None):
+    """
+    Build part of a raw cypher query for a node label.
+
+    :param node_label: a string containing a Neo4j node label
+    :kwarg uid_name: name of node's UniqueIdProperty
+    :kwarg uid: value of node's UniqueIdProperty
+    :return: a string containing node represented in raw cypher
+    """
+    if uid_name and uid:
+        return '({0}:{1} {{{2}:\"{3}\"}})'.format(node_label.lower(), node_label,
+                                                  uid_name.rstrip('_'), uid)
+    return '({0}:{1})'.format(node_label.lower(), node_label)
+
+
+def create_query(item, uid_name, uid, reverse=False):
+    """
+    Create a raw cypher query for a node label.
+
+    :param item: a neo4j node whose story is requested by the user
+    :param uid_name: name of node's UniqueIdProperty
+    :param uid: value of node's UniqueIdProperty
+    :param reverse: boolean value to specify the direction to proceed
+                    from current node corresponding to the story_flow
+    :return: a string containing raw cypher query to retrieve the story of an artifact from neo4j
+    """
+    # To avoid circular imports
+    from purview.models import story_flow
+
+    query = ''
+    curr_node_label = item.__label__
+
+    if reverse is True:
+        rel_label = 'backward_relationship'
+        node_label = 'backward_label'
+    else:
+        rel_label = 'forward_relationship'
+        node_label = 'forward_label'
+
+    while curr_node_label:
+
```

Comment:
Bike-shedding: Remove this new line since it makes the code below look separate from the while loop.